spec/integration_spec.rb

```diff
@@ -101,9 +101,76 @@ def cleanup(output)
 
   # SharedPrintOps - integration tests are in the respective SharedPrint class
 
-  xdescribe "Report" do
+  describe "Report" do
     before(:each) do
-      phctl("load", "cluster_file", fixture("cluster_2503661.json"))
+      # Data from spec/fixtures/cluster_2503661.json, without commitments
```

Comment:
An alternative could be loading a SQL dump, or at least a separate shared context with the insert statements.

Reply:
Moved to a support context.